products.js

```diff
@@ -30,20 +30,14 @@ const VIEWPORT_HEIGHT = 960;
  */
 function formatPrice(priceString){
   priceString = priceString.replace('$', '').replace(/([\s]|[^0-9$.-])/g, '');
-  if (priceString.includes('$')){
-    priceString = priceString.substr(priceString.indexOf('$') + 1);
-  }
-  return parseFloat(priceString);
+  return parseFloat(priceString.substr(priceString.indexOf('$') + 1));
```

Comment:
I'm sorry; I was imposing the Python replace semantics onto JS. In Python, it's always a global replace.